docs/user/available_components.rst

```diff
@@ -156,9 +156,18 @@ Typing
 - :doc:`components/patho_typing`: *In silico* pathogenic typing from raw
   illumina reads.
 
+- :doc:`components/seqsero2_reads`: Pipeline for Salmonella serotype determination
+  using SeqSero2 from raw reads.
+
+- :doc:`components/seqsero2_assembly`: Pipeline for Salmonella serotype determination
+  using SeqSero2 from genome assemblies.
+
 - :doc:`components/seq_typing`: Determines the type of a given sample from a set
   of reference sequences.
 
+- :doc:`components/stx_seqtyping_reads`: Gets Escherichia coli stx subtypes from raw
```

Comment:
I would change the name of this component to "stx_seqtyping" as the input information is available when you do "flowcraft build -L"

Reply:
I choose to place the _reads_ part in the component name because someone might ask for a similar component using assemblies (as it was implemented for SeqSero2)

Reply:
That makes no sense as in no other component the input is stated in the name, with the exception of components that are the sabe but have different inputs (your seqsero2 is an example of this). There is an ongoing projects to remove these situations entirely from flowcraft (see project https://github.com/assemblerflow/flowcraft/projects/3). I strongly recommend changing this.

Reply:
Done